conf/modules.config

```diff
@@ -68,20 +81,38 @@ params {
             publish_dir   = 'presto/08-assemblepairs'
             publish_files = [ 'log':'logs', 'tab':'tabs', 'log.txt':'logs' ]
             publish_by_id = true
-            args          = '--coord presto --rc tail --1f CONSCOUNT PRCONS --2f CONSCOUNT PRCONS'
+            args          = '--rc tail --1f CONSCOUNT PRCONS --2f CONSCOUNT PRCONS'
             args2         = '-f ID BARCODE SEQCOUNT PRIMER PRCOUNT PRCONS PRFREQ CONSCOUNT LENGTH OVERLAP ERROR PVALUE'
         }
+        'presto_assemblepairs_sans_umi' {
+            publish_dir   = 'presto/01-assemblepairs'
+            publish_files = [ 'log':'logs', 'tab':'tabs', 'log.txt':'logs' ]
+            publish_by_id = true
+            args          = '--rc tail'
+            args2         = '-f ID SEQCOUNT PRIMER PRCOUNT PRFREQ LENGTH OVERLAP ERROR PVALUE'
+        }
         'presto_parseheaders_collapse_umi' {
             publish_dir   = false
             subcommand    = 'collapse'
             args          = '-f CONSCOUNT --act min'
             publish_files = false
         }
+        'presto_parseheaders_collapse_sans_umi' {
+            publish_dir   = false
+            subcommand    = 'collapse'
+            args          = '-f CONSCOUNT --act min'
```

Comment:
I'm not sure about this here, as far as I understand it the `CONSCOUNT` value is the count of sequences having the same UMI barcode use to build a consensus. So without UMIs you should actually not have this field.

Reply:
Good catch! This was a vestigial step that snuck under my radar. Removed in d8dc9d0